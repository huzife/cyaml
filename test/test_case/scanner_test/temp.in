---
- 
  - aaa: abcde
      fghijk
  - bbb
- key1: 
        key11: ccc cccccc
        key12: ddddddd
