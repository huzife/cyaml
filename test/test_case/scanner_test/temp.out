(STREAM_START, )#
(DOC_START, ---)#
(BLOCK_SEQ_START, )#
(DASH, -)#
(BLOCK_SEQ_START, )#
(DASH, -)#
(BLOCK_MAP_START, )#
(KEY, aaa)#
(COLON, :)#
(SCALAR, abcde fghijk)#
(BLOCK_MAP_END, )#
(DASH, -)#
(SCALAR, bbb)#
(BLOCK_SEQ_END, )#
(DASH, -)#
(BLOCK_MAP_START, )#
(KEY, key1)#
(COLON, :)#
(BLOCK_MAP_START, )#
(KEY, key11)#
(COLON, :)#
(SCALAR, ccc cccccc)#
(KEY, key12)#
(COLON, :)#
(SCALAR, ddddddd)#
(BLOCK_MAP_END, )#
(BLOCK_MAP_END, )#
(BLOCK_SEQ_END, )#
(STREAM_END, )#
